### DISCUSSION TITLE
New component: Dropdown

### DISCUSSION THREAD

**Discussion by bigint** [heroui-inc/heroui#317] [Feature requests]
Something like this

<img width="306" alt="Screenshot 2021-09-12 at 4 29 33 PM" src="https://user-images.githubusercontent.com/69431456/132984967-e994573b-fe4e-4e07-85d6-5078c73f0f94.png">

**Comment by brettdoyle44** [comment]
Do you have an expected date for this feature @jrgarciadev ?

**Comment by max-programming** [comment]
Any alternatives suggested, please? Whilst it's not here yet.

**Comment by krystxf** [comment]
My favorite is headless UI https://headlessui.dev/react/menu

**Comment by max-programming** [comment]
@krystofex So do you suggest me using headless UI and Next UI together? Is it okay?

**Comment by krystxf** [comment]
It's easy to use, only drawback is that you will need to create custom styles for it.

**Comment by max-programming** [comment]
> It's easy to use, only drawback is that you will need to create custom styles for it.

Alright thanks @krystofex ! I'll give it a shot

**Comment by jrgarciadev** [comment]
Hey guys @bigint @krystofex  @max-programming did you see the NextUI `Dropdown` component? 🚀
https://nextui.org/docs/components/dropdown

**Reply by krystxf** [reply]
Looks amazing! 🚀
